doc/news/changes/minor/20171020GiovanniAlzetta

```diff
@@ -0,0 +1,3 @@
+New: Added BoundingBox::get_neighbor_type() which , given another BoundingBox, returns the enum type NeighborType. Added the function GridTools::compute_mesh_predicate_bounding_box() in grid_tools which returns a vector of BoundingBoxes containing the cells for which a given predicate is true.
```

Comment:
"which returns the neighboring relation between two bounding boxes. Added the function GridTools::compute_mesh_predicate_bounding_box() which returns a vector of BoundingBoxes covering the cells for which a given predicate is true."